src/index.spec.js

```diff
@@ -722,7 +738,9 @@ describe('reactElementToJSXString(ReactElement)', () => {
 
   it('reactElementToJSXString(<div> {false} </div>)', () => {
     expect(reactElementToJSXString(<div> {false} </div>)).toEqual(
-      '<div>\n    \n</div>'
+      `<div>
+  {'  '}
+</div>`
```

Comment:
I'm not fan of the output that we have in that case. But the previous one were not pretty... so I guess we have to look at it in a dedicated PR.